operator/charts/templates/clusterrole.yaml

```diff
@@ -27,6 +27,7 @@ rules:
   - podcliques/status
   - podcliquescalinggroups
   - podcliquescalinggroups/status
+  - clustertopologies
```

Comment:
why should `Grove` operator have `create, delete, deletecollection, patch and update` roles for `clustertopologies`?